components/inputs/test/input-checkbox.html

```diff
@@ -113,9 +113,30 @@
 						expect(input.indeterminate).to.be.true;
 					});
 
+					it('should bind "checked" attribute to input property', async() => {
+						elem.setAttribute('checked', 'checked');
+						await elem.updateComplete;
+						expect(input.checked).to.be.true;
+					});
+
+					it('should bind "checked" property to input property', async() => {
+						elem.checked = true;
+						await elem.updateComplete;
+						expect(input.checked).to.be.true;
+					});
+
+					it('should continue to bind "checked" property after interaction', async() => {
+						input.click();
+						expect(input.checked).to.be.true;
+						expect(elem.checked).to.be.true;
+						await elem.updateComplete;
+						elem.checked = false; // eslint-disable-line require-atomic-updates
+						await elem.updateComplete;
+						expect(input.checked).to.be.false;
+					});
```

Comment:
@tylergee I simplified the test you had a bit to just replicate the issue once. If you noticed a good reason to flip it back and forth a couple times I can make it more complex, but this seemed to do the trick.

Reply:
No good reason to leave the flip-flop in there, nope. I just did that to fully reassure myself that things were working properly in both directions. For testing that the bug is fixed, just the simple case is good.